lovely/seal.toml

```diff
@@ -9,15 +9,35 @@ priority = 0
 [patches.regex]
 target = 'card.lua'
 pattern = '''
-(?<indent>[\t ]*)if seal_type > 0.75 then card:set_seal\('Red'\)
-[\t ]*elseif seal_type > 0.5 then card:set_seal\('Blue'\)
-[\t ]*elseif seal_type > 0.25 then card:set_seal\('Gold'\)
-[\t ]*else card:set_seal\('Purple'\)
-[\t ]*end'''
+(?<indent>[\t ]*)local seal_rate = 10
+[\n\t ]*local seal_poll = pseudorandom\(pseudoseed\('stdseal'..G.GAME.round_resets.ante\)\)
+[\n\t ]*if seal_poll > 1 - 0.02\*seal_rate then
+[\n\t ]*local seal_type = pseudorandom\(pseudoseed\('stdsealtype'..G.GAME.round_resets.ante\)\)
+[\n\t ]*if seal_type > 0.75 then card:set_seal\('Red'\)
+[\n\t ]*elseif seal_type > 0.5 then card:set_seal\('Blue'\)
+[\n\t ]*elseif seal_type > 0.25 then card:set_seal\('Gold'\)
+[\n\t ]*else card:set_seal\('Purple'\)
+[\n\t ]*end
+[\n\t ]*end'''
 position = 'at'
 line_prepend = '$indent'
 payload = '''
-card:set_seal(SMODS.Seal.rng_buffer[math.ceil(seal_type*#SMODS.Seal.rng_buffer) or 1])'''
+card:set_seal(poll_seal(nil, 10))'''
+
+# Card:calculate_joker()
+[[patches]]
+[patches.regex]
+target = 'card.lua'
+pattern = '''
+(?<indent>[\t ]*)local seal_type = pseudorandom\(pseudoseed\('certsl'\)\)
```

Comment:
Consistency: why does this exclude modded seals?

Reply:
I was unsure whether or not to change this, do we want it to respect modded seals or act as vanilla behaviour in all circumstances?